addons/web/static/src/scss/report.scss

```diff
@@ -64,3 +64,11 @@ div#total {
         }
     }
 }
+
+/* Wkhtmltopdf doesn't handle flexbox properly */
```

Comment:
- Use `//` comments which are sass comment instead of `/*` which actually produce css ;)